modules/ROOT/attachments/admin-api.yaml

```diff
@@ -281,6 +281,57 @@ paths:
         200:
           description: Delete user success
           content: {}
+  /security/oidc/whoami:
+    get:
+      tags: 
+        -  Authentication
+      summary: Verify identity of current user and resolve to a principal.
+      description: Obtain the principal and details of the JWT passed in the bearer token.
+      operationId: oidc_whoami   
+      responses:
+        200:
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/oidc_whoami_response'            
+        401:
+          description: Unauthorized
+          content: {}      
+  /security/oidc/keys/cache_invalidate:
+    post:
+      tags: 
+        -  Authentication
+      summary: Reload the keys from the Identity Provider
+      description: Flush the JWK cache and force reload keys.
+      operationId: oidc_keys_cache_invalidate
+      responses: 
+        200:
+          description: Keys reload successful
+          content: {}
+        500:
+          description: Keys reload failed
+          content:
+            application/json:
+              schema: 
+                $ref: '#/components/schemas/oidc_keys_cache_invalidate_error_response'
+  /security/oidc/revoke:
+    post: 
+      tags: 
+        -  Authentication
+      summary: Reload the keys from the Identity Provider and disconnect clients
```

Comment:
```suggestion
      summary: Reload the keys from the Identity Provider and disconnect clients.
```

Reply:
added end punctuation because the first summary I came to had it. Regardless of the convention you choose, just be consistent.